docs/UserGuide.md

```diff
@@ -523,16 +556,31 @@ Hence, you type in the command `add-lesson c\1 d\TUESDAY st\10:00 et\11:00 v\COM
 
 #### Editing a lesson : `edit-lesson`
 
-Edits an existing lesson in the application.
+If you entered wrong details pertaining to your lesson and would like to change it, you can make use of this command.
+
+Format: `edit-lesson c\CLASS_INDEX l\LESSON_INDEX [d\DAY] [st\START_TIME] [et\END_TIME] [v\VENUE]`
+
+For example:
+* You made an error while entering the `CS2103T Tutorial` lesson. It should be `Tueday` and not `Thursday`. You type the command `edit-lesson c\1 l\1 d\Tuesday` and press <kbd>Enter</kbd>.
+
+  ![Edit lesson before](images/ugimages/EditLessonBefore.png)
 
-Format: `edit-lesson c/CLASS_INDEX l/LESSON_INDEX [d/DAY] [st/START_TIME] [et/END_TIME] [v/VENUE]`
+* The day of the lesson has been changed from `Thursday` to `Tuesday`.
 
-* Edits the lesson at the specified `CLASS_INDEX` + `LESSON_INDEX`.
+  ![Edit lesson after](images/ugimages/EditLessonAfter.png)
+
+Constraints:
+* The index refers to the index number shown in the displayed class list.
```

Comment:
The meaning of CLASS_INDEX and LESSON_INDEX are explained in the glossary. Perhaps we don't need to explain here again?